app/src/main/java/edu/uc/kovaciad/slicetracker/dto/SliceFile.kt

```diff
@@ -37,6 +39,7 @@ data class SliceFile (@PrimaryKey var sfid: Int,
                       @ColumnInfo(name = "resinLiftHeight") var resinLiftHeight : Int = 0,
                       @ColumnInfo(name = "numberOfLayers") var numberOfLayers : Int = 0,
                       @ColumnInfo(name = "filamentNozzleThickness") var filamentNozzleThickness : Double = 0.0,
+                      //Time should be changed with GregorianCalander (I dont want to mess up your DB so i wont make this change)
```

Comment:
On: " //Time should be changed with GregorianCalander (I dont want to mess up your DB so i wont make this change)"

You're in a separate branch.  You can make any change you want, without messing up the database.  That's why we use branches.  :)